controllers/apartments.rb

```diff
@@ -1,14 +1,23 @@
-# index
+# index route
 get '/apartments' do
+	@apartments = Apartment.all
   erb :"apartments/index"
 end
 
-# new
+# new route
 get '/apartments/new' do
   erb :"apartments/new"
 end
 
-# show
+# show route
 get '/apartments/:id' do
+	@apartment = Apartment.find(params[:id])
+	@tenants = Tenant.where(apartment_id: @apartment[:id])
```

Comment:
Oh and while your use of `[]` syntax isn't wrong either, `@apartment.id` would be preferred over `@apartment[:id]`